src/components/Notifications/Messages.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+export const incorrectEmail = 'Неправильний формат електронної пошти'; 
+
+export const emptyInput = (name?:string)=>{
+    return name ? `Поле ${name} є обов'язковим` : `Поле є обов'язковим`;
+}; 
+
+export const incorrectPhone = 'Дане поле не є номером телефону'; 
+
+export const onlyPositiveNumber = `Поле не може бути від'ємним`; 
+
+export const minLength = (len:number)=>{
+    return `Мінімальна довжина - ${len} символів`
+}; 
+
+export const maxLength = (len:number)=>{
+    return `Максимальна довжина - ${len} символів`
+}; 
+
+export const minCount = (len:number)=>{
+    return `Мінімальна довжина - ${len} символів`
+}; 
+
+export const maxCount = (len:number)=>{
+    return `Максимальна довжина - ${len} символів`
+}; 
```

Comment:
You write 2 methods: maxCount and maxLenght with the same returning string

Reply:
Thanks! I've changed it.